bcda/auth/client/ssas.go

```diff
@@ -56,7 +56,7 @@ func NewSSASClient() (*SSASClient, error) {
 
 	var timeout int
 	if timeout, err = strconv.Atoi(conf.GetEnv("SSAS_TIMEOUT_MS")); err != nil {
-		log.SSAS.Info("Could not get SSAS timeout from environment variable; using default value of 500.")
+		log.SSAS.Info(errors.Wrap(err, "Could not get SSAS timeout from environment variable; using default value of 500."))
```

Comment:
Out of curiosity, why aren't we able to do something like `log.SSAS.Error` here? 

Also -- I've noticed `.Error` and `Errorf` -- what's the difference?

Reply:
Oh! Good question — I didn't look at what method was being used here. In general I use the following definitions for each log level:

- INFO: Informational
- WARN: Not a show-stopper, but something out-of-the-ordinary 
- ERROR: A show-stopper (unexpected error in the system, stops the rest of the function)

In this case, I'll change it to `log.SSAS.Warn` since it's not expected to happen but we have a fallback/default value.

Reply:
Oh and your second question! `Error` is for basic messages and `Errorf` accepts multiple arguments for formatting.

```
log.SSAS.Error("I did something wrong")
log.SSAS.Errorf("I did something wrong for organization %s", organization.id) 
```